lib/common.sh

```diff
@@ -625,6 +625,24 @@ common::sha () {
   which shasum >/dev/null 2>&1 && shasum -a$algo $file | awk '{print $1}'
 }
 
+# Check for and source security layer
```

Comment:
That's fine.  I'm not married to security_layer.  I just wanted something that indicated a protected include that was generic enough and could be used by another organization if needed as well.  I suspect we'll get as many answers as the number of people we'll ask.   Let's try a few more @spxtr @pwittrock